resources/views/users/index.blade.php

```diff
@@ -14,15 +14,15 @@
         </div>
     </div>
 
-    @if (count($admins) > 0)
+    @if ($admins->count())
         @include('users.partials._table_users', ['users' => $admins, 'role' => 'Admins'])
     @endif
 
-    @if (count($staff) > 0)
+    @if ($staff->count())
         @include('users.partials._table_users', ['users' => $staff, 'role' => 'Staff'])
     @endif
 
-    @if (count($contestants) > 0)
+    @if ($contestants->count())
```

Comment:
Ahh, yeah. The `Collection` class implements PHP's [`Countable`](@if ($contestants->count())) interface but that (frustratingly!) doesn't apply for `empty()`... only for `count()`. Suggestion rescinded! :stuck_out_tongue_winking_eye: